src/main/java/pl/poznan/put/cs/scenariochecker/controller/strategy/CountStepsScenarioStrategy.java

```diff
@@ -0,0 +1,36 @@
+package pl.poznan.put.cs.scenariochecker.controller.strategy;
```

Comment:
we could put our ScenarioStrategy and classes that implement it in another package next to 'controller' package. For example one named 'transformations'.  what do you think about that?